<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1371</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>1061</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Поворот</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Масштабирование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Xc:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>P:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Yc:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="menu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Изменение фигур</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Треугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ромб</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Шестиугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5-ти конечная звезда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6-ти конечная звезда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8-ми конечная звезда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
